gcalcli/cli.py

```diff
@@ -20,6 +20,8 @@
 # Everything you need to know (Google API Calendar v3): http://goo.gl/HfTGQ #
 #                                                                           #
 # ######################################################################### #
+import truststore
+truststore.inject_into_ssl()
```

Comment:
Linter doesn't like this. I'd move the `inject_into_ssl()` call down into main() and add a small comment about what it does, then move the import down to be sorted.